Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -128,274 +244,186 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
      */
     public var sendsNotifications: Bool = true
```

Comment:
This class has a number of Boolean properties that could be characterized as feature flags, such as `sendsNotifications`, `snapsUserLocationAnnotationToRoute`, `automaticallyAdjustsStyleForTimeOfDay`, and `showsSpeedLimits`. Should we group them under a single header like “Toggling Optional Functionality” or scatter them among less vague sections about the subject areas? If the latter, it might be worth creating a separate guide listing these feature flags.